<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QWidget" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>69</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_0">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/icon/play_on.png</normaloff>:/icon/play_on.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/icon/next.png</normaloff>:/icon/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+Right</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_voice">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
border: 1px solid #7a4d5f;
background: #C0C0C0;
height: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
}
 
QSlider::sub-page:horizontal {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
    stop:0 #B1B1B1, stop:1 #c4c4c4);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #fc6984, stop: 1 #040f1b);
border: 1px solid #4A708B;
height: 10px;
border-radius: 2px;
}
 
QSlider::add-page:horizontal {
background: #575757;
border: 0px solid #777;
height: 10px;
border-radius: 2px;
}
 
QSlider::handle:horizontal 
{
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, 
    stop:0.6 #7a4d5f, stop:0.778409 rgba(255, 255, 255, 255));
 
    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}
 
QSlider::handle:horizontal:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.6 #be6570, 
    stop:0.778409 rgba(255, 255, 255, 255));
 
    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}
 
QSlider::sub-page:horizontal:disabled {
background: #00009C;
border-color: #999;
}
 
QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}
 
QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}
</string>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_voice">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/icon/volume.png</normaloff>:/icon/volume.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icon/wall_paper.png</normaloff>:/icon/wall_paper.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_screen">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icon/screen_full.png</normaloff>:/icon/screen_full.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_upend">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icon/upend.png</normaloff>:/icon/upend.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_speed">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icon/speed_1.0.png</normaloff>:/icon/speed_1.0.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_floder">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icon/folder.png</normaloff>:/icon/folder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>31</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_set">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/icon/set.png</normaloff>:/icon/set.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="slider_frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
border: 1px solid #7a4d5f;
background: #C0C0C0;
height: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
}
 
QSlider::sub-page:horizontal {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
    stop:0 #B1B1B1, stop:1 #c4c4c4);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #fc6984, stop: 1 #040f1b);
border: 1px solid #4A708B;
height: 10px;
border-radius: 2px;
}
 
QSlider::add-page:horizontal {
background: #575757;
border: 0px solid #777;
height: 10px;
border-radius: 2px;
}
 
QSlider::handle:horizontal 
{
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, 
    stop:0.6 #7a4d5f, stop:0.778409 rgba(255, 255, 255, 255));
 
    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}
 
QSlider::handle:horizontal:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.6 #be6570, 
    stop:0.778409 rgba(255, 255, 255, 255));
 
    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}
 
QSlider::sub-page:horizontal:disabled {
background: #00009C;
border-color: #999;
}
 
QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}
 
QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>00:00:00 / 00:00:00</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_play</tabstop>
  <tabstop>pushButton_next</tabstop>
  <tabstop>pushButton_voice</tabstop>
  <tabstop>slider_voice</tabstop>
  <tabstop>pushButton_speed</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
